Cargo.toml

```diff
@@ -32,6 +32,7 @@ members = [
     "./apps/only-peers",
     "./apps/gen-ext",
     "./apps/visited",
+    "./apps/blockchain",
```

Comment:
can we try a better name maybe?

Reply:
that will be in separate repo. here just for testing